daemon/oci_linux.go

```diff
@@ -480,9 +480,10 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
+			data := c.HostConfig.Tmpfs[m.Destination]
```

Comment:
@cpuguy83 WDYT about this - isn't is weird to retrieve the data for tmpfs mount here from HostConfig instead of having it already in volume.MountPoint?